src/App.js

```diff
@@ -1,40 +1,41 @@
-import logo from './logo.svg';
 import React, { Component } from 'react';
 import './App.css';
 import movieData from './data'
 import MovieContainer from './MovieContainer'
+import MovieDetailsCard from './MovieDetailsCard'
 
 class App extends Component {
   constructor() {
     super()
-    this.state = movieData
+    this.state = {
+      movies: movieData.movies,
+      displayMovieDetails: null
```

Comment:
I'd be interested in discussing this default value of null here. I know Leta said to be mindful of the data types we set these to, but I'm not sure I fully understand what null is doing here. Not saying it's wrong, I just need a sesh to wrap my head around it 😅